user/session_test.go

```diff
@@ -168,3 +171,189 @@ func TestAPILimit_Duration(t *testing.T) {
 		assert.Equal(t, expectedDuration, limit.Duration())
 	})
 }
+
+func TestAPILimitIsEmpty(t *testing.T) {
```

Comment:
Also move to /tests/<area>, let's not extend gw scope if we can avoid it.

Reply:
I believe this can stay here since this is a helper function.

Reply:
Correct, missed that it's in user scope, all good